src/main/java/uk/gov/homeoffice/digital/sas/person/model/Person.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.homeoffice.digital.sas.person.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
+import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+import uk.gov.homeoffice.digital.sas.person.enums.TermsAndConditions;
+
+@Resource(path = "persons")
+@Entity(name = "person")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@NoArgsConstructor
+@Getter
+@Setter
+public class Person extends BaseEntity {
+
+  @NotNull(message = "Person Version should not be empty")
+  @Column(name = "version")
+  private Integer version;
+
+  @NotNull(message = "Person First Name should not be empty")
+  @Column(name = "first_name")
+  private String firstName;
```

Comment:
Perhaps we should include a reasonable length validation here just in case someone decides to input unrealistic/out of bound string lengths. (last name too)

Reply:
True. Thanks. I've added the same length as it is in the SQL table create query. 50. Should be more than enough but not overkill. It could be less than 50 but this one is quite a safe length I guess.

Reply:
yep that sounds reasonable.